### DISCUSSION TITLE
Clause order

### DISCUSSION THREAD

**Discussion by spauldhaliwal** [simolus3/drift#2367] [Q&A] [answered]
Does clause order matter when building queries from dart? For example, if using a limit with a where clause in sql, we are supposed to put the limit **after** the where clause etc...

Does drift automatically order the clauses correctly or should we take care in how we order them ourselves?

**Comment by FaFre** [comment, marked as answer]
You can just write your queries and drift will take care of it. 

You also can still write pure sql in .drift files and make use of the code generator.
